advanced/am-pattern-4/values.yaml

```diff
@@ -0,0 +1,299 @@
+      mysql: true
+      # Enable NFS dynamic provisioner for Kubernetes
+      nfsServerProvisioner: true
+
+    # Persisted and shared runtime artifacts for API Manager
+    # See official documentation (from https://apim.docs.wso2.com/en/latest/install-and-setup/setup/reference/common-runtime-and-configuration-artifacts/#persistent-runtime-artifacts)
+    persistentRuntimeArtifacts:
+      # Kubernetes Storage Class to be used to dynamically provision the relevant Persistent Volumes
+      # Only persistent storage solutions supporting ReadWriteMany access mode are applicable (https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes)
+      # Defaults to Kubernetes Storage Class generated using the NFS Server Provisioner (https://github.com/helm/charts/tree/master/stable/nfs-server-provisioner)
+      storageClass: &storage_class "nfs"
+
+      # Define capacities for persistent runtime artifact directories which are shared between instances of the relevant API Manager profile
+      sharedArtifacts:
+        capacity:
+          # For execution plans shared between the Traffic Manager profile instances
+          executionPlans: 20M
+          # For synapse artifacts of APIs shared between the Gateway profile instances
+          synapseConfigs: 50M
+
+      # Persistent runtime artifacts for Apache Solr-based indexing
+      apacheSolrIndexing:
+        # Indicates if persistence of the runtime artifacts for Apache Solr-based indexing is enabled
+        # By default, this is disabled
+        enabled: false
+        # Define capacities for persistent runtime artifact directories
+        capacity:
+          # For persisting the H2 based local Carbon database file
+          carbonDatabase: 50M
+          # For persisting the indexed data
+          solrIndexedData: 50M
+
+    am:
+      # Container image configurations
+      # If a custom image must be used, uncomment 'dockerRegistry' and provide its value
+      # dockerRegistry: ""
+      imageName: "wso2am"
+      imageTag: "3.1.0"
+      # Refer to the Kubernetes documentation on updating images (https://kubernetes.io/docs/concepts/containers/images/#updating-images)
+      imagePullPolicy: Always
+
+      resources:
+        # These are the resource recommendations for running WSO2 API Management product profiles with profile optimization
+        # Resource configurations defined here are applicable for Gateway and Key Manager profile deployments, which use profile optimization
+        requests:
+          memory: "1Gi"
+          cpu: "1000m"
+        limits:
+          memory: "2Gi"
+          cpu: "2000m"
+
+      # API Manager's Gateway specific configurations
+      gateway:
+        internal:
+          # Hostname for Internal Gateway profile
+          hostname: "internal.gateway.am.wso2.com"
+          # Ingress class for internal Gateway
+          ingress:
+            # Annotations for the Internal Gateway service ingress
```

Comment:
@chathuraswso2 please create an issue for this task and also appreciate if you can actually fix this in a separate PR.

And most importantly, test it properly.

Reply:
Fix in https://github.com/wso2/kubernetes-apim/pull/444/commits/5d99c7a0cc84924f01b31dbbba09b9169c67e61d